README.md

```diff
@@ -19,6 +19,8 @@ Please consider donating, if you like my work
 ```
 $ npm install json-nested-replace
 ```
+## NPM Version Badge
+[![NPM Version](https://img.shields.io/npm/v/json-nested-replace.svg)](https://www.npmjs.com/package/json-nested-replace)
```

Comment:
You can move this at the top of the file.
Right below the title and above `[![Build Status]`

Reply:
Done